universal-login-relayer/test/integration/ethereum/MessageExecutor.test.ts

```diff
@@ -1,37 +1,55 @@
-import {SignedMessage, TEST_ACCOUNT_ADDRESS} from '@unilogin/commons';
+import {SignedMessage, TEST_ACCOUNT_ADDRESS, Message, TEST_TOKEN_PRICE_IN_ETH, TEST_GAS_PRICE_IN_TOKEN, TEST_GAS_PRICE} from '@unilogin/commons';
+import {getMockedGasPriceOracle} from '@unilogin/commons/testutils';
 import {emptyMessage} from '@unilogin/contracts/testutils';
 import {expect} from 'chai';
 import {loadFixture} from 'ethereum-waffle';
-import {Contract, providers, Wallet} from 'ethers';
+import {Contract, providers, Wallet, utils} from 'ethers';
 import {bigNumberify} from 'ethers/utils';
 import MessageExecutor from '../../../src/integration/ethereum/MessageExecutor';
 import {getTestSignedMessage} from '../../testconfig/message';
 import {basicWalletContractWithMockToken} from '../../fixtures/basicWalletContractWithMockToken';
 import MessageMemoryRepository from '../../mock/MessageMemoryRepository';
 import {setupWalletContractService} from '../../testhelpers/setupWalletContractService';
+import {GasTokenValidator} from '../../../src/core/services/validators/GasTokenValidator';
 
 describe('INT: MessageExecutor', () => {
   let messageExecutor: MessageExecutor;
   let signedMessage: SignedMessage;
   let provider: providers.Provider;
   let wallet: Wallet;
   let walletContract: Contract;
+  let message: Message;
   const validator = {
-    validate: async () => {},
+    validate: () => {},
```

Comment:
We should keep type of fn
```suggestion
    validate: () => Promise.resolve() ,
```